app/src/main/java/org/kinecosystem/kinit/util/SupportUtil.kt

```diff
@@ -12,22 +12,35 @@ import org.kinecosystem.kinit.repository.UserRepository
 
 object SupportUtil {
 
+    fun openEmailFeedback(context: Context, userRepository: UserRepository) {
+        val emailIntent = Intent(Intent.ACTION_SENDTO, Uri.fromParts(
+                "mailto", context.resources.getString(R.string.feedback_email_address), null))
+        if (context.packageManager != null && emailIntent.resolveActivity(context.packageManager) != null) {
+            emailIntent.putExtra(Intent.EXTRA_SUBJECT, context.resources.getString(R.string.feedback_email_subject))
+            val message = context.resources.getString(R.string.feedback_email_body_template)
+            emailIntent.putExtra(Intent.EXTRA_TEXT, message)
+            context.startActivity(emailIntent)
+        } else {
+            showSupportDialog(context, userRepository.userId())
+        }
```

Comment:
You can refactor into one openEmail method passing it the  email address + resource ids for subject, body template 
Also we should have different text on showSupportDialog for support/feedback.